src/components/VulnerabilityEdgePrompt.js

```diff
@@ -82,7 +82,6 @@ const VulnerabilityEdgePrompt = () => {
         canSelect={false}
         canEditPairStatus={false}
         canManageColumns={false}
-        linkToCustomerPortal
```

Comment:
If this was the last usage of this parameter, I guess it can be completely removed. wdyt folks? @mkholjuraev